src/main/scala/io/treeverse/clients/LakeFSContext.scala

```diff
@@ -15,19 +15,34 @@ object LakeFSContext {
   val LAKEFS_CONF_JOB_REPO_NAME_KEY = "lakefs.job.repo_name"
   val LAKEFS_CONF_JOB_COMMIT_ID_KEY = "lakefs.job.commit_id"
 
-  def newRDD(sc: SparkContext, repoName : String, commitID: String): RDD[(Array[Byte], WithIdentifier[Catalog.Entry])] = {
+  def newRDD(
+      sc: SparkContext,
+      repoName: String,
+      commitID: String
+  ): RDD[(Array[Byte], WithIdentifier[Catalog.Entry])] = {
     val conf = new Configuration(sc.hadoopConfiguration)
     conf.set(LAKEFS_CONF_JOB_REPO_NAME_KEY, repoName)
     conf.set(LAKEFS_CONF_JOB_COMMIT_ID_KEY, commitID)
     if (StringUtils.isBlank(conf.get(LAKEFS_CONF_API_URL_KEY))) {
-      throw new InvalidJobConfException("%s must not be empty".format(LAKEFS_CONF_API_URL_KEY))
+      throw new InvalidJobConfException(
+        "%s must not be empty".format(LAKEFS_CONF_API_URL_KEY)
```

Comment:
While we're at it,
```suggestion
        s"${LAKEFS_CONF_API_URL_KEY} must not be empty",
```
might be easier on the eyes than explicitly calling a _method_ on `String`.  Also AFAIU the Scala Style Guide says to do this, https://docs.scala-lang.org/style/declarations.html.